policykit/policyengine/models.py

```diff
@@ -335,34 +324,6 @@ def make_api_call(cls, community, values, call, action=None, method=None):
         res = community.make_call(call, values=values, action=action, method=method)
         return res
 
-class GenericPolicy(models.Model):
```

Comment:
These models are removed, so there should be migration files for deleting those tables from the database. Can you run `makemigrations` and commit the result?